controllers/replication.storage/volumereplication_controller.go

```diff
@@ -614,6 +651,30 @@ func (r *VolumeReplicationReconciler) enableReplication(vr *volumeReplicationIns
 	return nil
 }
 
+// getVolumeReplicationInfo gets volume replication info.
+func (r *VolumeReplicationReconciler) getLastSyncTime(vr *volumeReplicationInstance) (*timestamppb.Timestamp, error) {
```

Comment:
IMO we should not be doing this. Currently, we have only one parameter in the `GetInfo` this looks fine, If we add more parameters we need to write a new function for it. My suggestion is to return a complete response `infoResponse` and use it in caller